linera-core/src/data_types.rs

```diff
@@ -58,6 +58,20 @@ impl BlockHeightRange {
     }
 }
 
+impl Display for BlockHeightRange {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if let Some(limit) = self.limit {
+            if limit == 0 && limit == 1 {
+                write!(f, "{}", self.start)
```

Comment:
For `0` I'd write "empty" or something.

Reply:
`0` should be disallowed but I didn't want to change the API of `BlockHeightRange`.